tests/ext/read_c_configuration.phpt

```diff
@@ -6,6 +6,7 @@ DD_TRACE_MEMORY_LIMIT=9999
 --FILE--
 
 <?php
+$bgs_should_be_enabled = PHP_VERSION_ID >= 50600;
```

Comment:
Nit: camelCase. :)

Reply:
For some reason that's hard for me to remember when switching from C to PHP >.<

Reply:
Fixed in acfb83efed.